### ISSUE TITLE
Using this program to play "osu!"

### ISSUE THREAD

**Issue by Tien-Bautzen** [arpruss/finger-draw#4]
hello, I want to use this program to play a video game called "osu!". most players are using a graphic tablet for aiming but since I don't have one I'm using my laptop touchpad. while I'm using my finger to aim I cannot use keyboard inputs (e.g. the Z and X keys) at the same time, which are necessary for clicking in the game. Can you make it somehow possible to allow keyboard inputs while I'm using my finger on the touchpad? Thank you!

**Comment by arpruss** [comment]
Can you type while touching the touchpad without using finger-draw?

**Comment by Tien-Bautzen** [comment]
Yes, it is possible for me to type with my left hand while touching the touchpad with my right hand at the same time (without finger-draw activated).

**Comment by arpruss** [comment]
What if you are clicking the touchpad at the same time?

**Comment by Tien-Bautzen** [comment]
It depends. The following happens when I have finger-draw turned on:

- Tapping on the touchpad doesn't seem to cause any problems with keyboard inputs.
- On the other hand, clicking with the touchpad buttons while typing doesn't seem to work.
- If one lets one finger on the touchpad (without lifting it up), one cannot type either.

Without finger-draw, all the abovementioned situations don't interfere with keyboard input.

**Comment by DorAntCr** [comment]
i have an idea: in your Windows Settings, try setting your touchpad sensitivity to the highest sensitivity. this is supposed to prevent accidental touches to the touchpad by disabling the touchpad whenever the keyboard is being typed on. maybe that will help, probably not
